test/widgets/test_thermal_zone.py

```diff
@@ -0,0 +1,20 @@
+import os
+from libqtile import widget
```

Comment:
You need a gap between these two imports.

Reply:
Done